runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingOverlay.java

```diff
@@ -78,7 +98,9 @@ public Dimension render(Graphics2D graphics)
 		panelComponent.getChildren().clear();
 		if (client.getLocalPlayer().getInteracting() != null
 			&& client.getLocalPlayer().getInteracting().getName().contains(FISHING_SPOT)
-			&& client.getLocalPlayer().getInteracting().getGraphic() != GraphicID.FLYING_FISH)
+			&& client.getLocalPlayer().getInteracting().getGraphic() != GraphicID.FLYING_FISH
+			&& client.getWidget(WidgetInfo.LEVEL_UP) == null
```

Comment:
I don't see a need to specifically check this case if we're adding an animation check.

Reply:
After level up dialogue the fishing animation continues for some time, and it was unclear to me if fishing was actually taking place, but I agree it's unneeded and it has been removed along with non-fishing animations.